rust/helm/hyperlane-agent/templates/scraper-external-secret.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: external-secrets.io/v1beta1
 kind: ExternalSecret
 metadata:
-  name: {{ include "agent-common.fullname" . }}-scraper-external-secret
+  name: {{ include "agent-common.fullname" . }}-scraper3-external-secret
```

Comment:
might make sense to make the version something that's added via a template

e.g. add a version to the scraper helm values, and then do something like this in `_helpers.tpl`:
```
{{- define "scraper.name" -}}
{{- if eq .Values.hyperlane.scraper.version 0 -}}
scraper
{{- else -}}
{{- printf "scraper%s" .Values.hyperlane.scraper.version -}}
{{- end }}
{{- end }}
```

And then you can do something like:
```
{{ include "agent-common.fullname" . }}-{{ include "scraper.name" }}-external-secret
```

Reply:
We would also have to change the helm release in the typescript. Not sure if it is worth it since it should not be changing regularly.

Reply:
I don't feel strongly, happy to keep as is if we don't see it changing in the future very often